meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.8" %}
```

Comment:
Why 0.8?

Reply:
In preparation for our conda release, we increased our version in \_\_init\_\_.py. See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html for more information.

Reply:
OK, but I thought it is 0.0.8 for xlum?

Reply:
That's correct :)